staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2549,6 +2550,21 @@ type PermittedHostDevices struct {
 	PciHostDevices []PciHostDevice `json:"pciHostDevices,omitempty"`
 	// +listType=atomic
 	MediatedDevices []MediatedHostDevice `json:"mediatedDevices,omitempty"`
+	// +listType=atomic
+	USB []USBHostDevice `json:"usb,omitempty"`
+}
+
+type USBHostDevice struct {
```

Comment:
Let's add the possibility of using external providers as PCI and Mediated devices provide. In the following PR

Reply:
Ok